***
# <font color=green size=10>CURSO DE ESTATÍSTICA - PARTE 1</font>
***

# <font color=green>1 CONHECENDO OS DADOS</font>
***

## <font color=green>1.1 Dataset do projeto</font>
***

### Pesquisa Nacional por Amostra de Domicílios - 2015

A <b>Pesquisa Nacional por Amostra de Domicílios - PNAD</b> investiga anualmente, de forma permanente, características gerais da população, de educação, trabalho, rendimento e habitação e outras, com periodicidade variável, de acordo com as necessidades de informação para o país, como as características sobre migração, fecundidade, nupcialidade, saúde, segurança alimentar, entre outros temas. O levantamento dessas estatísticas constitui, ao longo dos 49 anos de realização da pesquisa, um importante instrumento para formulação, validação e avaliação de políticas orientadas para o desenvolvimento socioeconômico e a melhoria das condições de vida no Brasil.

### Fonte dos Dados

https://ww2.ibge.gov.br/home/estatistica/populacao/trabalhoerendimento/pnad2015/microdados.shtm

### Variáveis utilizadas

> ### Renda
> ***

Rendimento mensal do trabalho principal para pessoas de 10 anos ou mais de idade.

> ### Idade
> ***

Idade do morador na data de referência em anos.

> ### Altura (elaboração própria)
> ***

Altura do morador em metros.

> ### UF
> ***

|Código|Descrição|
|---|---|
|11|Rondônia|
|12|Acre|
|13|Amazonas|
|14|Roraima|
|15|Pará|
|16|Amapá|
|17|Tocantins|
|21|Maranhão|
|22|Piauí|
|23|Ceará|
|24|Rio Grande do Norte|
|25|Paraíba|
|26|Pernambuco|
|27|Alagoas|
|28|Sergipe|
|29|Bahia|
|31|Minas Gerais|
|32|Espírito Santo|
|33|Rio de Janeiro|
|35|São Paulo|
|41|Paraná|
|42|Santa Catarina|
|43|Rio Grande do Sul|
|50|Mato Grosso do Sul|
|51|Mato Grosso|
|52|Goiás|
|53|Distrito Federal|

> ### Sexo
> ***

|Código|Descrição|
|---|---|
|0|Masculino|
|1|Feminino|

> ### Anos de Estudo
> ***

|Código|Descrição|
|---|---|
|1|Sem instrução e menos de 1 ano|
|2|1 ano|
|3|2 anos|
|4|3 anos|
|5|4 anos|
|6|5 anos|
|7|6 anos|
|8|7 anos|
|9|8 anos|
|10|9 anos|
|11|10 anos|
|12|11 anos|
|13|12 anos|
|14|13 anos|
|15|14 anos|
|16|15 anos ou mais|
|17|Não determinados|
||Não aplicável|

> ### Cor
> ***

|Código|Descrição|
|---|---|
|0|Indígena|
|2|Branca|
|4|Preta|
|6|Amarela|
|8|Parda|
|9|Sem declaração|

#### <font color='red'>Observação</font>
***
> Os seguintes tratamentos foram realizados nos dados originais:
> 1. Foram eliminados os registros onde a <b>Renda</b> era inválida (999 999 999 999);
> 2. Foram eliminados os registros onde a <b>Renda</b> era missing;
> 3. Foram considerados somente os registros das <b>Pessoas de Referência</b> de cada domicílio (responsável pelo domicílio).

### Importando pandas e lendo o dataset do projeto

https://pandas.pydata.org/

In [ ]:
import pandas as pd

In [ ]:
dados = pd.read_csv('dados.csv')

In [ ]:
type(dados)

pandas.core.frame.DataFrame

In [ ]:
dados

UF  Sexo  Idade  Cor  Anos de Estudo  Renda    Altura
0      11     0     23    8              12    800  1.603808
1      11     1     23    2              12   1150  1.739790
2      11     1     35    8              15    880  1.760444
3      11     0     46    2               6   3500  1.783158
4      11     1     47    8               9    150  1.690631
...    ..   ...    ...  ...             ...    ...       ...
76835  53     1     46    2              11    812  1.687030
76836  53     0     30    4               7   1500  1.792934
76837  53     0     32    8              12   1300  1.830587
76838  53     0     57    8               4   1500  1.726344
76839  53     0     38    8               4    900  1.658305

[76840 rows x 7 columns]

In [ ]:
dados.head()

UF  Sexo  Idade  Cor  Anos de Estudo  Renda    Altura
0  11     0     23    8              12    800  1.603808
1  11     1     23    2              12   1150  1.739790
2  11     1     35    8              15    880  1.760444
3  11     0     46    2               6   3500  1.783158
4  11     1     47    8               9    150  1.690631

In [ ]:
dados.head(29)

UF  Sexo  Idade  Cor  Anos de Estudo  Renda    Altura
0   11     0     23    8              12    800  1.603808
1   11     1     23    2              12   1150  1.739790
2   11     1     35    8              15    880  1.760444
3   11     0     46    2               6   3500  1.783158
4   11     1     47    8               9    150  1.690631
5   11     1     34    8              12    790  1.637906
6   11     0     57    8              12   3150  1.570078
7   11     1     60    8              12   1700  1.608495
8   11     1     50    4              14   1800  1.780329
9   11     0     26    8              12   1150  1.793203
10  11     1     46    8              12   3000  1.787003
11  11     1     49    8               6   2500  1.712912
12  11     1     52    2               6   1200  1.736442
13  11     0     38    8              12   2000  1.750827
14  11     0     45    2              12   2000  1.780450
15  11     0     34    2              16   6000  1.711825
16  11     1     43    8              10   1050  1.789012
17  11     0     53    8              16   7000  1.673665
18  11     0     57    2              12   2000  1.715138
19  11     1     62    8              12   1150  1.765973
20  11     0     69    8               1   3030  1.746351
21  11     1     41    8              16   4000  1.660873
22  11     0     34    8               5    788  1.746048
23  11     0     25    8              12   1000  1.726496
24  11     0     33    8               9    800  1.778914
25  11     1     44    8               5    788  1.668638
26  11     0     33    2              16   1600  1.783771
27  11     0     65    2              12   1700  1.873445
28  11     0     33    8              14   2000  1.890250

## <font color=green>1.2 Tipos de dados</font>
***

**Por que idade é um dado quantitativo e anos de estudo é qualitativo?**

Na maioria dos casos, a idade é considerada um dado quantitativo (ou variável quantitativa) porque é uma medida numérica que pode ser expressa em termos de números inteiros ou decimais. A idade é uma quantidade mensurável que representa a quantidade de anos que uma pessoa viveu desde o seu nascimento. Ela pode ser usada em cálculos matemáticos, como médias, desvios padrão e outras operações numéricas. Por exemplo, você pode calcular a média de idades de um grupo de pessoas ou realizar cálculos estatísticos com base nas idades.

Por outro lado, anos de estudo são frequentemente considerados uma variável qualitativa (ou categórica) porque representam categorias ou grupos discretos de educação. Anos de estudo geralmente não são medidas contínuas, mas sim categorias que descrevem o nível de educação atingido por uma pessoa. Por exemplo, alguém pode ter concluído o ensino fundamental, o ensino médio, a graduação, a pós-graduação, etc. Essas categorias não são expressas em termos de números contínuos, e sim em termos de categorias qualitativas.

No entanto, é importante observar que a classificação de uma variável como quantitativa ou qualitativa pode variar dependendo do contexto e da análise estatística em questão. Em alguns casos, os anos de estudo podem ser tratados como uma variável quantitativa se forem usados como números contínuos em análises estatísticas específicas, como modelos de regressão. Portanto, a classificação de variáveis pode ser flexível e dependerá de como você deseja usá-las em sua análise.

### Variáveis qualitativas ordinais

► Variáveis que podem ser ordenadas ou hierarquizardas

In [ ]:
sorted(dados['Anos de Estudo'].unique())

[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17]

Uma **variável qualitativa ordinal**, também conhecida como variável ordinal, é um tipo de variável categórica que possui categorias ou grupos com uma ordem intrínseca. Em outras palavras, as categorias de uma variável ordinal podem ser classificadas ou ordenadas de acordo com algum critério, mas as diferenças entre as categorias não têm um significado numérico preciso. Em uma variável ordinal, a ordem das categorias é importante, mas a distância entre elas não é constante ou mensurável.

Exemplos comuns de variáveis ordinais incluem:

1. Níveis de educação: (por exemplo, Ensino Fundamental, Ensino Médio, Graduação, Pós-Graduação). Essas categorias têm uma ordem natural, mas a diferença entre cada nível não é uma medida numérica constante.

2. Classificações de satisfação do cliente: (por exemplo, Muito Insatisfeito, Insatisfeito, Neutro, Satisfeito, Muito Satisfeito). As classificações refletem uma ordem, mas não há uma diferença numérica precisa entre as categorias.

3. Níveis de concordância: (por exemplo, Discordo Totalmente, Discordo, Neutro, Concordo, Concordo Totalmente). As respostas refletem um grau de concordância em uma escala ordinal.

Variáveis ordinais são comuns em pesquisas sociais, pesquisas de opinião, avaliações de produtos e muitos outros contextos em que a classificação das categorias é importante, mas as diferenças precisas entre as categorias não podem ser quantificadas em termos de números exatos. Em análises estatísticas, as variáveis ordinais são frequentemente tratadas com técnicas específicas que levam em consideração a ordem das categorias, como análises de tendência central ou de frequência.

Uma **"ordem intrínseca" **se refere a uma ordem ou sequência que é natural ou inerente a um conjunto de elementos ou categorias, sem depender de uma atribuição arbitrária de valores numéricos. Essa ordem se baseia em características ou propriedades das categorias em si, e não em medidas quantitativas.

Em relação a variáveis qualitativas, como variáveis ordinais, a ordem intrínseca significa que as categorias têm uma relação de ordem significativa, mas não podem ser representadas numericamente de maneira contínua. Por exemplo, se você está avaliando a satisfação do cliente com opções como "Muito Insatisfeito", "Insatisfeito", "Neutro", "Satisfeito" e "Muito Satisfeito", existe uma ordem intrínseca onde "Muito Insatisfeito" é considerado pior do que "Insatisfeito", e assim por diante. No entanto, você não pode atribuir números específicos a essas categorias que representem uma diferença numérica precisa entre elas.

Em contraste, variáveis quantitativas, como a idade, possuem uma ordem intrínseca e também uma escala numérica significativa. Por exemplo, se alguém tem 30 anos de idade, sabemos que essa pessoa tem mais idade do que alguém com 25 anos, e a diferença de idade entre essas duas pessoas é de 5 anos.

Portanto, a ordem intrínseca refere-se à relação natural de ordem entre categorias ou elementos, que é qualitativa e não depende de valores numéricos contínuos. É importante considerar a ordem intrínseca ao analisar variáveis ordinais, pois ela pode ter implicações significativas na interpretação dos dados.

### Variáveis qualitativas nominais

► Variáveis que não podem ser ordenadas ou hierarquizardas

Uma variável **qualitativa nominal**, também conhecida como variável nominal, é um tipo de variável categórica que representa categorias ou grupos que não possuem uma ordem intrínseca. Em outras palavras, as categorias de uma variável nominal são distintas e independentes umas das outras, e não podem ser classificadas em uma ordem específica. As categorias em uma variável nominal são usadas para representar diferentes classes, tipos ou rótulos, mas não há relação numérica ou ordinal entre elas.

Exemplos comuns de variáveis nominais incluem:

1. Cores: (por exemplo, vermelho, azul, verde). As cores não têm uma ordem natural; são apenas rótulos distintos.

2. Estados civis: (por exemplo, solteiro, casado, divorciado, viúvo). Embora haja uma variedade de estados civis, eles não podem ser classificados em uma ordem numérica.

3. Tipos de veículos: (por exemplo, carro, caminhão, moto, bicicleta). Essas categorias representam diferentes tipos de veículos, mas não têm uma ordem específica.

Variáveis nominais são usadas para categorizar dados em grupos ou classes que não podem ser quantificados em termos de números ordenados. Em análises estatísticas, as variáveis nominais são tratadas usando técnicas adequadas, como tabelas de frequência, gráficos de barras e análises de contingência, que ajudam a entender a distribuição e as relações entre as categorias.

In [ ]:
sorted(dados['UF'].unique())

[11,
 12,
 13,
 14,
 15,
 16,
 17,
 21,
 22,
 23,
 24,
 25,
 26,
 27,
 28,
 29,
 31,
 32,
 33,
 35,
 41,
 42,
 43,
 50,
 51,
 52,
 53]

In [ ]:
sorted(dados['Sexo'].unique())

[0, 1]

In [ ]:
sorted(dados['Cor'].unique())

[0, 2, 4, 6, 8]

### Variáveis quantitativas discretas

► Variáveis que representam uma contagem onde os valores possíveis formam um conjunto finito ou enumerável.

**Variáveis quantitativas discretas** são um tipo de variável numérica que representa valores inteiros e específicos, onde não há possibilidade de valores intermediários entre dois pontos. Essas variáveis representam contagens ou números inteiros que são frequentemente resultado de uma contagem ou enumeração de elementos em uma determinada categoria. Em outras palavras, as variáveis quantitativas discretas não podem assumir valores fracionários ou contínuos, apenas valores inteiros.

Exemplos de variáveis quantitativas discretas incluem:

1. Número de filhos em uma família: Você pode ter 0, 1, 2, 3 filhos, mas não 1,5 filhos.

2. Número de carros em um estacionamento: Pode haver 0, 1, 2, 3, 4 carros, mas não 2,5 carros.

3. Número de alunos em uma sala de aula: Pode haver 10, 20, 30, 40 alunos, mas não 25,5 alunos.

4. Número de itens vendidos em uma loja em um determinado dia: Pode ser 100, 150, 200 itens, mas não 175,5 itens.

Variáveis quantitativas discretas são frequentemente usadas em estatísticas, análises de dados e pesquisas, e podem ser resumidas usando medidas como média, mediana, moda e desvio padrão, entre outras. Elas são distintas das variáveis quantitativas contínuas, que podem assumir qualquer valor dentro de um intervalo contínuo, como a altura ou o peso das pessoas.

In [ ]:
sorted(dados['Idade'].unique())

[13,
 14,
 15,
 16,
 17,
 18,
 19,
 20,
 21,
 22,
 23,
 24,
 25,
 26,
 27,
 28,
 29,
 30,
 31,
 32,
 33,
 34,
 35,
 36,
 37,
 38,
 39,
 40,
 41,
 42,
 43,
 44,
 45,
 46,
 47,
 48,
 49,
 50,
 51,
 52,
 53,
 54,
 55,
 56,
 57,
 58,
 59,
 60,
 61,
 62,
 63,
 64,
 65,
 66,
 67,
 68,
 69,
 70,
 71,
 72,
 73,
 74,
 75,
 76,
 77,
 78,
 79,
 80,
 81,
 82,
 83,
 84,
 85,
 86,
 87,
 88,
 89,
 90,
 91,
 92,
 94,
 95,
 97,
 99]

In [ ]:
dados.Idade.min()

13

In [ ]:
dados.Idade.max()

99

In [ ]:
print('De %s até %s anos' %(dados.Idade.min(), dados.Idade.max()))

De 13 até 99 anos


In [ ]:
print(f'De {dados.Idade.min()} até {dados.Idade.max()} anos')

De 13 até 99 anos


#### <font color='red'>Observação</font>
***
> A variável idade pode ser classificada de três formas distintas:
> 1. <b>QUANTITATIVA DISCRETA</b> - quando representa anos completos (números inteiros);
> 2. <b>QUANTITATIVA CONTÍNUA</b> - quando representa a idade exata, sendo representado por frações de anos; e
> 3. <b>QUALITATIVA ORDINAL</b> - quando representa faixas de idade.

### Variáveis quantitativas contínuas

► Variáveis que representam uma contagem ou mensuração que assumem valores em uma escala contínua (números reais).

**Variáveis quantitativas contínuas** são um tipo de variável numérica que pode assumir um número infinito de valores dentro de um intervalo contínuo. Essas variáveis representam medidas que podem ser expressas como números reais e que podem incluir valores fracionários ou decimais. Diferentemente das variáveis quantitativas discretas, que representam valores inteiros específicos, as variáveis contínuas podem assumir qualquer valor dentro de um intervalo.

Exemplos de variáveis quantitativas contínuas incluem:

1. Altura das pessoas: A altura pode variar de uma pessoa para outra e pode incluir valores decimais, como 1,65 metros ou 1,75 metros.

2. Peso das pessoas: O peso é uma variável contínua que pode ser medida com detalhes, incluindo valores decimais, como 70,5 kg ou 85,2 kg.

3. Temperatura ambiente: A temperatura pode variar continuamente e incluir valores decimais, como 20,5°C ou 23,7°C.

4. Tempo decorrido: O tempo decorrido, como a duração de um evento, pode ser expresso em unidades de tempo que incluem frações, como 2,5 horas ou 3,25 minutos.

Variáveis quantitativas contínuas são frequentemente usadas em análises estatísticas, modelagem matemática e pesquisas científicas, e são tratadas como valores numéricos em cálculos estatísticos. A interpretação e análise de variáveis contínuas geralmente envolvem estatísticas descritivas, como média, desvio padrão, quartis, e também técnicas de análise estatística avançada, como regressão e análise de variância.

In [ ]:
print(f'De {dados.Altura.min()} até {dados.Altura.max()} metros')

De 1.339244614 até 2.028496765 metros


In [ ]:
print('De %s até %s' % (dados['Altura'].min(), dados['Altura'].max()))

De 1.339244614 até 2.028496765


In [ ]:
print(f'Quantidade de Ocorrências {dados.Altura.count()}')
print(f'Média {dados.Altura.mean()}')
print(f'Mediana {dados.Altura.median()}')
print(f'Moda {dados.Altura.mode()}')
print(f'Desvio Padrão {dados.Altura.std()}')

Quantidade de Ocorrências 76840
Média 1.6995124540575741
Mediana 1.6993247325
Moda 0    1.568128
1    1.671225
2    1.681659
3    1.692977
4    1.708163
5    1.708370
6    1.753842
7    1.779073
8    1.796462
Name: Altura, dtype: float64
Desvio Padrão 0.07971743006825292


### Classificação de uma variável
<img src='https://caelum-online-public.s3.amazonaws.com/1177-estatistica-parte1/01/img001.png' width='70%'>

# <font color=green>2 DISTRIBUIÇÃO DE FREQUÊNCIAS</font>
***

O primeiro passo em um trabalho de análise é o conhecimento do comportamento das variáveis envolvidas no estudo. Utilizando técnicas estatísticas como as análises das <b>DISTRIBUIÇÕES DE FREQUÊNCIAS</b> e <b>HISTOGRAMAS</b> podemos avaliar melhor a forma como os fenômenos em estudo se distribuem.

## <font color=green>2.1 Distribuição de frequências para variáveis qualitativas</font>
***

### Método 1

https://pandas.pydata.org/pandas-docs/version/0.22/generated/pandas.Series.value_counts.html

### Método 2

https://pandas.pydata.org/pandas-docs/version/0.22/generated/pandas.crosstab.html

## <font color=green>2.2 Distribuição de frequências para variáveis quantitativas (classes personalizadas)</font>
***

### Passo 1 - Especificar os limites de cada classe

Utilizar a seguinte classificação:

<b>A</b> ► Acima de 20 SM

<b>B</b> ► De 10 a 20 SM

<b>C</b> ► De 4 a 10 SM

<b>D</b> ► De 2 a 4 SM

<b>E</b> ► Até 2 SM

onde <b>SM</b> é o valor do salário mínimo na época. Em nosso caso <b>R$ 788,00</b> (2015):

<b>A</b> ► Acima de 15.760

<b>B</b> ► De 7.880 a 15.760

<b>C</b> ► De 3.152 a 7.880

<b>D</b> ► De 1.576 a 3.152

<b>E</b> ► Até 1.576


### Passo 2 - Criar a tabela de frequências

https://pandas.pydata.org/pandas-docs/version/0.22/generated/pandas.cut.html

## <font color=green>2.3 Distribuição de frequências para variáveis quantitativas (classes de amplitude fixa)</font>
***

### Importando bibliotecas

http://www.numpy.org/

### Passo 1 - Difinindo o número de classes

#### Regra de Sturges

# $$k = 1 + \frac {10}{3}\log_{10}n$$

### Passo 2 - Criar a tabela de frequências

## <font color=green>2.4 Histograma</font>
***

O <b>HISTOGRAMA</b> é a representação gráfica de uma distribuição de frequências. É uma gráfico formado por um conjunto de retângulos colocados lado a lado, onde a área de cada retângulo é proporcional à frequência da classe que ele representa.

### Importando a biblioteca

https://seaborn.pydata.org/

# <font color=green>3 MEDIDAS DE TENDÊNCIA CENTRAL</font>
***

## DataFrame de exemplo

## <font color=green>3.1 Média aritmética</font>
***

É representada por $\mu$ quando se refere à população e por $\bar{X}$ quando se refere à amostra

# $$\mu = \frac 1n\sum_{i=1}^{n}X_i$$

onde

$n$ = número de observações (registros)

$X_i$ = valor da i-ésima observação (registro)

## <font color=green>3.2 Mediana</font>
***

Para obtermos a mediana de uma conjunto de dados devemos proceder da seguinte maneira:
1. Ordenar o conjunto de dados;
2. Identificar o número de observações (registros) do conjunto de dados ($n$);
3. Identicar o elemento mediano:

> Quando $n$ for ímpar, a posição do elemento mediano será obtida da seguinte forma:


# $$Elemento_{Md} = \frac{n+1}2$$

> Quando $n$ for par, a posição do elemento mediano será obtida da seguinte forma:


# $$Elemento_{Md} = \frac{n}2$$

4. Obter a mediana:

> Quando $n$ for ímpar:


# $$Md = X_{Elemento_{Md}}$$

> Quando $n$ for par:


# $$Md = \frac{X_{Elemento_{Md}} + X_{Elemento_{Md}+1}}2$$
***

### Exemplo 1 - n ímpar

<img src='https://caelum-online-public.s3.amazonaws.com/1177-estatistica-parte1/01/img002.png' width='40%' style="float:left">

### Exemplo 2 - n par

<img src='https://caelum-online-public.s3.amazonaws.com/1177-estatistica-parte1/01/img003.png' width='50%' style="float:left">

### Obtendo a mediana em nosso dataset

## <font color=green>3.3 Moda</font>
***

Pode-se definir a moda como sendo o valor mais frequente de um conjunto de dados. A moda é bastante utilizada para dados qualitativos.

### Obtendo a moda em nosso dataset

## <font color=green>3.4 Relação entre média, mediana e moda</font>
***

<img src='https://caelum-online-public.s3.amazonaws.com/1177-estatistica-parte1/01/img004.png' width='80%'>

### Avaliando a variável RENDA

***

### Avaliando a variável ALTURA

***

### Avaliando a variável ANOS DE ESTUDO

# <font color=green>4 MEDIDAS SEPARATRIZES</font>
***

## <font color=green>4.1 Quartis, decis e percentis</font>
***

Há uma série de medidas de posição semelhantes na sua concepção à mediana, embora não sejam medidas de tendência central. Como se sabe, a mediana divide a distribuição em duas partes iguais quanto ao número de elementos de cada parte. Já os quartis permitem dividir a distribuição em quatro partes iguais quanto ao número de elementos de cada uma; os decis em dez partes e os centis em cem partes iguais.

## <font color=green>4.2 Box-plot</font>
***

O box plot dá uma idéia da posição, dispersão, assimetria, caudas e dados discrepantes (outliers). A posição central é dada pela mediana e a dispersão por $IIQ$. As posições relativas de $Q1$, $Mediana$ e $Q3$ dão uma noção da simetria da distribuição. Os comprimentos das cauda são dados pelas linhas que vão do retângulo aos valores remotos e pelos valores atípicos.

<img src='https://caelum-online-public.s3.amazonaws.com/1177-estatistica-parte1/01/img005.png' width='65%'>

<img src='https://caelum-online-public.s3.amazonaws.com/1177-estatistica-parte1/01/img006.png' width='80%'>

# <font color=green>5 MEDIDAS DE DISPERSÃO</font>
***

Embora as medidas de posição forneçam uma sumarização bastante importante dos dados, elas podem não ser suficientes para caracterizar conjuntos distintos, especialmente quando as observações de determinada distribuição apresentarem dados muito dispersos.

## <font color=green>5.1 Desvio médio absoluto</font>
***


# $$DM = \frac 1n\sum_{i=1}^{n}|X_i-\bar{X}|$$


## <font color=green>5.2 Variância</font>
***

### Variância

A variância é construída a partir das diferenças entre cada observação e a média dos dados, ou seja, o desvio em torno da média. No cálculo da variância, os desvios em torno da média são elevados ao quadrado.

### Variância populacional

# $$\sigma^2 = \frac 1n\sum_{i=1}^{n}(X_i-\mu)^2$$

### Variância amostral

# $$S^2 = \frac 1{n-1}\sum_{i=1}^{n}(X_i-\bar{X})^2$$

## <font color=green>5.3 Desvio padrão</font>
***

Uma das restrições da variância é o fato de fornecer medidas em quadrados das unidades originais - a variância de medidas de comprimento, por exemplo, é em unidades de área. Logo, o fato de as unidades serem diferentes dificulta a comparação da dispersão com as variáveis que a definem. Um modo de eliminar essa dificuldade é considerar sua raiz quadrada.

### Desvio padrão populacional

# $$\sigma = \sqrt{\frac 1n\sum_{i=1}^{n}(X_i-\mu)^2} \Longrightarrow \sigma = \sqrt{\sigma^2}$$

### Desvio padrão amostral

# $$S = \sqrt{\frac 1{n-1}\sum_{i=1}^{n}(X_i-\bar{X})^2} \Longrightarrow S = \sqrt{S^2}$$